### DISCUSSION TITLE
Configuration for OS Open Zoomstack

### DISCUSSION THREAD

**Discussion by vidhav** [ldproxy/ldproxy#1061] [General]
Is the configuration for the OS Open Zoomstack ([https://demo.ldproxy.net/zoomstack/](https://demo.ldproxy.net/zoomstack/)) available anywhere? It would be very interesting to see how it is put together.

**Comment by cportele** [comment]
@vidhav - Yes, all configurations on demo.ldproxy.net are documented at https://github.com/ldproxy/demo including OS Open Zoomstack.

**Reply by vidhav** [reply]
Thank you, @cportele! It was right there, all along 👍
